src/IFX007T-Motor-Control.h

```diff
@@ -65,21 +67,21 @@ class IFX007TMotorControl
         void    begin(void);
         void    end(void);
 
-        void    setUniDirMotorSpeed(uint8_t motor, uint8_t dutycycle);          //For Unidirectional motors; Parameters: motor can be 0, 1 or 2, dutycycle can be 0 - 255
-        void    setBiDirMotorSpeed(bool direction, uint8_t dutycycle);          //For Bidirectional motors; Parametrs: direction can be 0 or 1, dutycycle can be 0 - 255
+        void    setUniDirMotorSpeed(uint8_t motor, uint8_t dutycycle);                      //For Unidirectional motors
+        void    setBiDirMotorSpeed(uint8_t motor, bool direction, uint8_t dutycycle);       //For Bidirectional motors
         void    configureBLDCMotor(BLDCParameter MyParameters);  
         void    setBLDCmotorRPMspeed(bool direction, uint16_t desired_rpmSpeed);
-        void    setBLDCDutyCyclespeed(bool direction, uint8_t dutycycle);                         
+        void    setBLDCDutyCyclespeed(bool direction, uint8_t dutycycle);
+        void    setHallBLDCmotorRPMspeed(bool direction, uint16_t desired_rpmSpeed, bool FieldWeakening);
```

Comment:
Are all these new API functions added to the keywords.txt? 
The setBiDirMotorSpeed() API has been changed, meaning, the major version needs to be changed. 
This needs to be changed then in the library.properties. 
Overloading the function would be possibility as well (if the previous function still makes sense to keep). Then  the next version could only be a minor increase.

Reply:
Ok I restored the old BiDirMotorSpeed function and the new one is now overloaded. Hardware tested again as well. I also inreased the Version number in the library.properties to 1.1.0 is this ok?